### DISCUSSION TITLE
[Feature] Sync, character and language features

### DISCUSSION THREAD

**Discussion by xiongsp** [immich-app/immich#14686] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

# Sync Features(Mobile)
1. Allow users to choose to sync local photos with cloud photos. Currently, only backup is supported, meaning that when two phones have different photos, they can only be backed up to the cloud simultaneously, but local photos and cloud photos cannot be synchronized. Just like iCloud, allow users to choose a switch to keep photos on multiple devices synchronized with the cloud.

# Character Features(Mobile, Web)
1. Customizing the cover photo for people, similar to the Photos app on iOS.
2. A button that can focus on the face

# Language Features(Mobile, Web)
Currently, the multilingual support for search and maps is not good. In search, you cannot use Chinese to search for 'Lion'; in maps, place names with the same pronunciation are grouped together as one, even though they are actually two different places.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Like the template requests, please keep each post to only one request and search for duplicates.
